### PR TITLE
fix(select list): update select list default value

### PR DESCRIPTION
**Pull request by lavanya-bmw**
## Description

Updated default value functionality to select list

## Why

As default value was not displayed and not selected from select list

## Issue

Ref: CPLP-3473

## Checklist

Please delete options that are not relevant.

- [x] I have followed the [contributing guidelines](https://github.com/eclipse-tractusx/portal-assets/blob/main/developer/Technical%20Documentation/Dev%20Process/How%20to%20contribute.md#commit-and-pr-guidelines)
- [ ] I have performed [IP checks](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-04#checking-libraries-using-the-eclipse-dash-license-tool) for added or updated 3rd party libraries
- [ ] I have created and linked IP issues or requested their creation by a committer
- [x] I have performed a self-review of my own code
- [x] I have successfully tested my changes locally
- [ ] I have added tests that prove my changes work
- [ ] I have checked that new and existing tests pass locally with my changes
- [ ] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: src/components/basic/SelectList/SelectList.stories.tsx [hunk 1]
```diff
@@ -79,6 +79,7 @@ SelectList.args = {
   clearText: 'clear',
   noOptionsText: 'No Options',
   popperHeight: 0, // 0 = auto size
+  defaultValue: countries[0],
```
Comment (jjeroch) [inline src/components/basic/SelectList/SelectList.stories.tsx:82]: @lavanya-bmw what does this default value "countries" stand for? I can not connect it to the task ticket

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 2.1.22
+
+- Update default value functionality to select list
```
Comment (jjeroch) [inline CHANGELOG.md:5]: @lavanya-bmw please be more specific
I suggest we always name first of all the component - in this case "SelectList component..." and afterwards the change. Is it really an update or did you actually added the functionality that the select list has an defect value set?

Last but not least: has this change an impact of current used selectList components?

**Review by jjeroch — CHANGES_REQUESTED** [review]

File: src/components/basic/SelectList/SelectList.stories.tsx (hunk 1 above)
Comment (lavanya-bmw) [reply src/components/basic/SelectList/SelectList.stories.tsx:82]: @jjeroch this is stories file which displays a default value to the items(countries) from line 70. This change is to see the output in shared components.

File: CHANGELOG.md (hunk 2 above)
Comment (lavanya-bmw) [reply CHANGELOG.md]: Since default value was not working for selectlist, this fix was made

**Review by jjeroch — APPROVED** [review]

**Review by evegufy — APPROVED** [review]

**Pull request merged**